blog/2023-03-14-nushell_0_77.md

```diff
@@ -190,6 +190,60 @@ Signatures:
   <nothing> | http -> <string>
 ```
 
+## Examples results are now shown in `help` pages ([amtoine](https://github.com/nushell/nushell/pull/8189))
+
+When commands define expected results for their examples, the output is now shown just below the associated examples.
+
+Let's take the `merge` command as an example!
+
+Before the change, the output of `help merge` would give
+
+```bash
+Examples:
+  Add an 'index' column to the input table
+  > [a b c] | wrap name | merge ( [1 2 3] | wrap index )
+
+  Merge two records
+  > {a: 1, b: 2} | merge {c: 3}
+
+  Merge two tables, overwriting overlapping columns
+  > [{columnA: A0 columnB: B0}] | merge [{columnA: 'A0*'}]
+```
+
+Now, it gives the output of all the example commands:
+
+```bash
+Examples:
+  Add an 'index' column to the input table
+  > [a b c] | wrap name | merge ( [1 2 3] | wrap index )
+  ╭───┬──────╮
+  │ # │ name │
+  ├───┼──────┤
+  │ 1 │ a    │
+  │ 2 │ b    │
+  │ 3 │ c    │
+  ╰───┴──────╯
+
+  Merge two records
+  > {a: 1, b: 2} | merge {c: 3}
+  ╭───┬───╮
+  │ a │ 1 │
+  │ b │ 2 │
+  │ c │ 3 │
+  ╰───┴───╯
+
+  Merge two tables, overwriting overlapping columns
+  > [{columnA: A0 columnB: B0}] | merge [{columnA: 'A0*'}]
+  ╭───┬─────────┬─────────╮
+  │ # │ columnA │ columnB │
+  ├───┼─────────┼─────────┤
+  │ 0 │ A0*     │ B0      │
+  ╰───┴─────────┴─────────╯
+```
+
+> :bulb: **Note**  
+> the website has been modified accordingly with the same outputs.
+
 Make sure to browse the help messages of these commands. They contain fully functional examples thanks to pointing at www.example.com.
```

Comment:
This should remain with the `http` commands

Reply:
very good catch, thanks :relieved: 

fixed back in b20cafb075232bb6e7b49bc6edee85674e25b620